src/views/components/define-signatory-info/signatory-details.njk

```diff
@@ -0,0 +1,107 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
```

Comment:
This file is a partial. It renders the HTML for a single signatory. If there are multiple signatories, then this partial will be rendered multiple times. HTML `id` and `name` are appended with the signatory ID to ensure uniqueness and easily map a signatory in session to the submitted data.